srcCpp/RTIDDSImpl.cxx

```diff
@@ -112,6 +112,9 @@ void RTIDDSImpl<T>::Shutdown()
         if (!registry->unregister("wh", NULL, NULL)) {
             //printf("failed to unregister wh\n");
         }
+        if (!registry->unregister("_shmem", NULL, NULL)) {
```

Comment:
We need to check if this is > 2.4.11 and only do it for that, right?

Reply:
I have already tested for 2.4.11. There is no need to make a difference
Any error would be printed.

Reply:
Done